src/app/static/src/app/store/metadata/metadata.ts

```diff
@@ -38,22 +38,32 @@ export const metadataGetters = {
             return [];
         }
 
-        const selectedDataType = rootState.surveyAndProgram.selectedDataType;
+        const sap = rootState.surveyAndProgram;
+        const selectedDataType = sap.selectedDataType;
 
         let metadataForType: Metadata | null = null;
+        let dataIndicators: FilterOption[] = [];
         switch (selectedDataType) {
             case (DataType.ANC):
                 metadataForType = plottingMetadata.anc;
+                dataIndicators = sap.anc?.filters.indicators || [];
                 break;
             case (DataType.Program):
                 metadataForType = plottingMetadata.programme;
+                dataIndicators = sap.program?.filters.indicators || [];
                 break;
             case (DataType.Survey):
                 metadataForType = plottingMetadata.survey;
+                dataIndicators = sap.survey?.filters.indicators ||[];
                 break;
         }
 
-        return (metadataForType && metadataForType.choropleth) ? metadataForType.choropleth.indicators : [];
+        const unfiltered = (metadataForType && metadataForType.choropleth) ? metadataForType.choropleth.indicators : [];
```

Comment:
```suggestion
        const unfiltered = (metadataForType?.choropleth) ? metadataForType.choropleth.indicators : [];
```
I think this would be the same and a little shorter